### PR TITLE
LPS-199625

### PR DESCRIPTION
**Pull request by MicaelleSilvaa**
Ticket: https://liferay.atlassian.net/browse/LPS-199625

### CODE REVIEW

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava [hunk 1]
```diff
@@ -14,6 +14,8 @@ import java.util.Date;
 
 import com.liferay.portal.kernel.service.ServiceContextThreadLocal;
 
+import com.liferay.portal.kernel.cookies.CookiesManagerUtil;
+
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava:18]: @MicaelleSilvaa, please sort the imports in alphabetical order :wink:

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava [hunk 2]
```diff
@@ -77,6 +79,19 @@ public class UpgradeCatchAllCheck {
 		_commerceAccountHelper.getCurrentCommerceAccount(null);
 	}
 
+	public void method(
+		HttpServletRequest httpServletRequest, String name) {
+		CookieUtil.get(httpServletRequest, "test123");
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava:84]: @MicaelleSilvaa, when a method signature is declared in 2 or more lines then we need to separate it from the method's content using a blank line, please see an example [here](https://github.com/natocesarrego/liferay-portal/blob/9f0279d3781597b87b32ec8350a9236a8da778a4/modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava#L64-L67).

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava [hunk 3]
```diff
@@ -77,6 +79,19 @@ public class UpgradeCatchAllCheck {
...
+		CookiesManagerUtil.getCookieValue(name, httpServletRequest);
+	}
+
+	public void method(
+		HttpServletRequest httpServletRequest, String name,
+		boolean toUpperCase) {
+		CookieUtil.get(httpServletRequest, "test123", toUpperCase);
+		CookiesManagerUtil.getCookieValue(name, httpServletRequest, toUpperCase);
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava:92]: @MicaelleSilvaa, please see my previous comment

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeCatchAllCheck.testjava [hunk 4]
```diff
@@ -62,6 +62,19 @@ public class UpgradeCatchAllCheck {
 		_commerceAccountHelper.getCurrentCommerceAccount(null);
 	}
 
+	public void method(
+		HttpServletRequest httpServletRequest, String name) {
+		CookieUtil.get(httpServletRequest, "test123");
+		CookieUtil.get(httpServletRequest, name);
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeCatchAllCheck.testjava:68]: @MicaelleSilvaa, isn't it the same case? Should we handle the 2 calls above really differently? Can you please clarify?

Honestly, I believe it seems to be a bug in UpgradeCatchAll, after all the 2 calls above have the same number of parameters with the exact same types

**Review by natocesarrego — CHANGES_REQUESTED** [review]

**Comment by natocesarrego** [comment]
@MicaelleSilvaa, can you rebase to fix this conflict?

Thanks

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava (hunk 1 above)
Comment (MicaelleSilvaa) [reply modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava:18]: The import is sorted by the json object

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava (hunk 2 above)
Comment (MicaelleSilvaa) [reply modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava:84]: Resolved

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava (hunk 3 above)
Comment (MicaelleSilvaa) [reply modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava:92]: Resolved

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeCatchAllCheck.testjava (hunk 4 above)
Comment (MicaelleSilvaa) [reply modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeCatchAllCheck.testjava:68]: In the method signature we declare the variable name and type, this is an error case when it is not possible to identify the type/name of the variable. We always use defaults: "null, test123 or 123".

**Pull request closed without merging**